### ISSUE TITLE
Berryboot wont recognize a USB Stick

### ISSUE THREAD

**Issue by Jan200101** [maxnet/berryboot#319]
I used 2 USB sticks already both with a custom OS that is already formated for BerryBoot but for some reason BerryBoot says that I cant recognize any Usb Stick or anyother External Harddrive.

I am using a Raspberry 3.

**Comment by maxnet** [comment]
Are you trying to use an USB stick as installation target in this screen?

![bb2](https://cloud.githubusercontent.com/assets/1123701/17717882/f146d5fe-6410-11e6-814f-17d69080b84f.png)

Or are you trying to install custom images stored on an USB stick.
If the latter make sure the USB stick is formatted FAT32 or ext4
Do not use NTFS, exFAT, etc.

**Comment by Jan200101** [comment]
Found the Error. For some reason my USB Stick auto formated itself to NTFS when it should have beein FAT32 because that is its standart.

Thanks for your help

**Comment by Hsad1644** [comment]
## TL;DR
* USB stick (**OR** SD for usb boot)
   * FAT32/EXT4 formatted **only**
   * must have [berryboot compatible images](https://berryboot.alexgoldcheidt.com/images/)
   * can have custom OS imgs which are [prepared appropriately](https://www.berryterminal.com/doku.php/berryboot/adding_custom_distributions)

Another thing to mention, the berryboot page has its own images and besides also mentions [a way to convert a custom OS .img](https://www.berryterminal.com/doku.php/berryboot/adding_custom_distributions) file into something within that 4GB limit for FAT32. 

### The berryboot images are available here https://berryboot.alexgoldcheidt.com/images/

The usb stick *is* being mounted (sometimes it doesn't for whatever reasons). Just that the .img files you have on them are not compatible with berryboot. Something to do with SquashFS .img files. Not well versed with that so won't comment further.

I came across a video by leepspvideo on YT and he was using images from the aforementioned site. Sure enough, it worked.
